CHANGE_LOG

```diff
@@ -1,3 +1,92 @@
+Version 0.39.0
+--------------
+
+Here are the highlights for the Numba 0.39.0 release.
+
+* This is the first version that supports Python 3.7.
+* With help from Intel, we have added fixed the SVML support (related issues
+  #2938, #2998, #3006).
+* List has gained support for containing reference-counted types like NumPy
+  arrays and `list`.  Note, list still can't hold heterogeneous types.
+* We have done a significant change to the internal calling-convention,
+  which should be transparent to most users, to allow for a future feature to
+  jump back into python-mode from a nopython-mode function.  This also fixed
+  limitation to `print` that disables it usage from nopython functions that
+  are deep in the callstack.
+* For CUDA GPUs support, we added a `__cuda_array_interface__` following the
+  NumPy array interface specification to allow Numba to consume externally
+  defined device array.  We have opened a corresponding pull request to CuPy to
+  test out the concept and be able to use a CuPy GPU array.
+
+Enhancements:
+
+* PR #3041: This adds @needs_blas to tests that use BLAS
```

Comment:
descending order to be consistent with last 6ish releases? Probably doesn't matter :)